docs/modules/ROOT/pages/faq.adoc

```diff
@@ -1,25 +1,25 @@
 [[faq]]
-= FAQ, Best Practices, and "How do I...?"
+== FAQ, Best Practices, and "How do I...?"
 
 This section covers the following content:
 
-* <<faq.wrap-blocking>>
-* <<faq.chain>>
-* <<faq.monoThen>>
-* <<faq.retryWhen>>
-* <<faq.exponentialBackoff>>
-* <<faq.thread-affinity-publishon>>
-* <<faq.mdc>>
+* xref:faq.adoc#faq.wrap-blocking[How Do I Wrap a Synchronous, Blocking Call?]
+* xref:faq.adoc#faq.chain[I Used an Operator on my `Flux` but it Doesn't Seem to Apply. What Gives?]
+* xref:faq.adoc#faq.monoThen[My `Mono` `zipWith` or `zipWhen` is never called]
+* xref:faq.adoc#faq.retryWhen[How to Use `retryWhen` to Emulate `retry(3)`?]
```

Comment:
I just noticed the C.4. section has no link to it here (it is also missing in the current documentation), can you please add it?

Reply:
Fixed in https://github.com/reactor/reactor-core/pull/3806/commits/539a40486790b75db40d9781ebf99a576c85403a